### PR TITLE
Automatically add labels to PR

### PR DESCRIPTION
**Pull request by kin0992**
#### Motivation and Context

We want to add some labels to the PR's so we can easily see what the changes are about.

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/labeler.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+frontend: apps/nextjs-website/**/*
+internal: packages/**/*
+docs: docs/**/*
+ci: .github/workflows/**/*
+infra: .infrastructure/**/*
```
Comment (datalek) [inline .github/labeler.yaml:5]:
```suggestion
frontend: apps/nextjs-website/**/*
packages: packages/**/*
docs: docs/**/*
workflows: .github/workflows/**/*
infra: .infrastructure/**/*
```

**Review by datalek — COMMENTED** [review]
WDYT?

File: .github/labeler.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+frontend: apps/nextjs-website/**/*
```
Comment (kin0992) [inline .github/labeler.yaml:1]:
```suggestion
website: apps/nextjs-website/**/*
```

**Review by datalek — APPROVED** [review]
🧢

**Pull request merged**